### ISSUE TITLE
Remove space in powershell command

### ISSUE THREAD

**Issue by tibcoplord** [openshift/windows-machine-config-bootstrapper#208]
At https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/tools/ansible/docs/azure/azure-with-windows-server.md we have -

```
... ConfigureR emotingForAnsible ...
```

This should be -

```
... ConfigureRemotingForAnsible ...
```

ie we have an extra space

**Comment by tibcoplord** [comment]
There is also an extra space in -

```
$ oc create -f https://gist.githubusercontent.com/suhanime/683ee7b5a2f55c11e3a26a4223170582/raw/d89 3db98944bf615fccfe73e6e4fb19549a362a5/WinWebServer.yaml -n default
```

**Comment by aravindhp** [comment]
We will fix this when we get a chance but is of lower priority at this point.

cc: @codyhoag

**Comment by codyhoag** [comment]
@tibcoplord thanks for reporting! I fixed in #212.

**Comment by aravindhp** [comment]
/close
Fixed by @codyhoag in #212
